panoramix/models.py

```diff
@@ -765,15 +768,34 @@ def get_pydruid_client(self):
             self.broker_endpoint)
         return cli
 
+    def parse_config(self, config_file):
+        params = {}
+        conf = ConfigFactory.parse_file(config_file)
+        params['name'] = conf['name']
+        params['dimensions'] = conf['dimensions']
+        params['metrics_spec'] = conf['metrics_spec']
+        params['replication'] = conf.get('replication', 2)
+        params['sql'] = conf.get('batch_ingestion.sql', None)
+        params['num_shards'] = conf.get('batch_ingestion.num_shards', 1)
+        params['ts_column'] = conf['batch_ingestion.ts_column']
+        params['sources'] = conf['batch_ingestion.sources']
+        return params
+
     def refresh_datasources(self):
         endpoint = (
             "http://{self.coordinator_host}:{self.coordinator_port}/"
             "{self.coordinator_endpoint}/datasources"
         ).format(self=self)
 
-        datasources = json.loads(requests.get(endpoint).text)
-        for datasource in datasources:
-            DruidDatasource.sync_to_db(datasource, self)
+        if not config.get('AIROLAP_CONFIGS', None):
+            datasources = json.loads(requests.get(endpoint).text)
```

Comment:
I think we should leave this block unchanged, so that non-airolap datasources work the same way as before.